libs/dissect/src/main/java/org/elasticsearch/dissect/DissectParser.java

```diff
@@ -309,7 +329,7 @@ private DissectPair(DissectKey key, String delimiter) {
             this.delimiter = delimiter;
         }
 
-        private DissectKey getKey() {
+        DissectKey getKey() {
```

Comment:
I don't quite see the need for the change in visibility here -- am I just missing something?

Reply:
Not really, just a leftover, I'll remove it.